wrench/src/main.rs

```diff
@@ -682,7 +695,7 @@ fn render<'a>(
 
         #[cfg(feature = "gfx")]
         {
-            let dims = window.get_window().get_inner_size().unwrap();
+            let dims = window.get_window().unwrap().get_inner_size().unwrap();
```

Comment:
I think we can use the `get_inner_size` of the `WindowWrapper` here, instead of the underlying window.